toolkit/smart-contracts/offchain/src/d_param/mod.rs

```diff
@@ -63,9 +63,9 @@ pub async fn upsert_d_param<
 	await_tx: &A,
 ) -> anyhow::Result<Option<MultiSigSmartContractResult>> {
 	let ctx = TransactionContext::for_payment_key(payment_signing_key, ogmios_client).await?;
-	let (validator, policy) = crate::scripts_data::d_parameter_scripts(genesis_utxo, ctx.network)?;
-	let validator_address = validator.address_bech32(ctx.network)?;
-	let validator_utxos = ogmios_client.query_utxos(&[validator_address]).await?;
+	let script_data = crate::scripts_data::d_parameter_scripts(genesis_utxo, ctx.network)?;
```

Comment:
I would go with "scripts", because for sure validator and policy are different scripts.